dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -658,6 +683,43 @@ protected HttpComplianceDeserializer(@Nullable Class<?> vc) {
         }
     }
 
+    private static class UriComplianceSerializer extends StdSerializer<UriCompliance> {
```

Comment:
As this is the third pair of similar (de)serializers, maybe we should provide utility classes for such 'simple' enum (de)serializers in the `dropwizard-util` or `dropwizard-jackson` module. @dropwizard/committers Any opinions on that?

Reply:
Sorry, have not had a chance.

Reply:
Taking a quick look, one issue with using the enum serializer/deserializer is that while `UriCompliance`, `CookieCompliance` and `HttpCompliance` all contain `Violation` enums and look a lot like enums, they aren't actually enums.

Reply:
@denis-yuen Thanks for the input. But I think the classes could use an other common (de)serializer implementation because of their similatity. If Jackson provides no such implementation, we could implement one ourselves into the `dropwizard-util` module with public access or into the `dropwizard-jetty` module with package-private access.